release/cli/pkg/bundles/bottlerocket.go

```diff
@@ -24,7 +25,7 @@ import (
 	releasetypes "github.com/aws/eks-anywhere/release/cli/pkg/types"
 )
 
-func GetBottlerocketHostContainersBundle(r *releasetypes.ReleaseConfig, eksDReleaseChannel string, imageDigests map[string]string) (anywherev1alpha1.BottlerocketHostContainersBundle, error) {
+func GetBottlerocketHostContainersBundle(r *releasetypes.ReleaseConfig, eksDReleaseChannel string, imageDigests sync.Map) (anywherev1alpha1.BottlerocketHostContainersBundle, error) {
```

Comment:
Can we pass an interface with the `Load` method here instead of `sync.Map`? I think we can create a struct that implements it, and the type assertion can be done there.